tests/test_support.py

```diff
@@ -253,3 +254,16 @@ def test_validate_paths(data_contract_directory: str):
 
     with pytest.raises(FileNotFoundError):
         validate_paths(paths, raise_exception=True)
+
+
+@pytest.mark.parametrize(
+    "input_uri, expected_uri",
+    [
+        ("/opt/data/data.csv", "file"),
+        ("C:/data/data.csv", "file"),
+        ("http://somserver/file.dat", "http")
```

Comment:
nit: did you mean to make this someserver?  (missing e)

Reply:
I will update. Thanks!